src/integrationTest/java/uk/gov/hmcts/reform/translate/controllers/DictionaryControllerIT.java

```diff
@@ -121,31 +126,103 @@ void shouldReturn200WhenLegalRequestIsSubmitted() throws Exception {
             final Map<String, String> expectedTranslations = Map.of("English Phrase 2", "Translated Phrase 2");
 
             mockMvc.perform(post(TRANSLATIONS_URL)
-                                .contentType(MediaType.APPLICATION_JSON_VALUE)
-                                .accept(MediaType.APPLICATION_JSON_VALUE)
+                                .contentType(APPLICATION_JSON_VALUE)
+                                .accept(APPLICATION_JSON_VALUE)
                                 .content("{\"phrases\": [\"English Phrase 2\"]}"))
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$.translations", equalTo(expectedTranslations)));
         }
+
+        @Test
+        @Sql(scripts = {DELETE_TRANSLATION_TABLES_SCRIPT})
+        void shouldTestConcurrentAddToDictionaryViaTranslateEndpoint() throws Exception {
+            final ExecutorService executorService = Executors.newFixedThreadPool(4);
+            try {
+                // GIVEN
+                Collection<Callable<ResultActions>> callables = new ArrayList<>();
+                IntStream.rangeClosed(1, 4)
+                    .forEach(i -> callables.add(createRequestTranslationCallable()));
+
+                // WHEN
+                final List<Future<ResultActions>> taskFutureList = executorService.invokeAll(callables);
+
+                // THEN
+                assertThatResultsContainsConstraintViolation(taskFutureList);
+            } finally {
+                Objects.requireNonNull(executorService).shutdown();
+            }
+        }
+    }
+
+    private Callable<ResultActions> createRequestTranslationCallable() {
+        return new Callable<>() {
+            final String payload = "{\"phrases\": [\"English phrase 2\"]}";
+
+            public ResultActions call() throws Exception {
+                return mockMvc.perform(post(TRANSLATIONS_URL)
+                                           .contentType(APPLICATION_JSON_VALUE)
+                                           .accept(APPLICATION_JSON_VALUE)
+                                           .content(payload));
+            }
+        };
     }
 
     @Nested
     @DisplayName("Put Dictionary")
     class PutDictionary {
+        private static final String SERVICE_AUTHORIZATION = "ServiceAuthorization";
```

Comment:
Feel free to use static import of:
```
uk.gov.hmcts.reform.translate.security.SecurityUtils.SERVICE_AUTHORIZATION
```

Which in turn should probably equal a static import of: 
```
uk.gov.hmcts.reform.authorisation.filters.ServiceAuthFilter.AUTHORISATION
```

Reply:
... ie.e. import of `uk.gov.hmcts.reform.authorisation.filters.ServiceAuthFilter.AUTHORISATION` may cause confsuion with other auth header `org.springframework.http.HttpHeader.AUTHORIZATION`

Reply:
Adjusted in 6745da253740d36b7e1fc787428db9a78b52c4fe